scene/gui/rich_text_label.cpp

```diff
@@ -556,13 +556,14 @@ int RichTextLabel::_process_line(ItemFrame *p_frame, const Vector2 &p_ofs, int &
 									w += font->get_char_size(c[i], c[i + 1]).x;
 								}
 
-								if (c[i] == '\t') {
-									visible = false;
-								}
-
 								if (visible) {
 									if (selected) {
-										cw = font->get_char_size(fx_char, c[i + 1]).x;
+										// Check to give tab correct cw so tab also looks selected when selecting text
+										if (fx_char == 9) {
```

Comment:
Why use a magic number instead of the `'\t'` literal?

Reply:
@akien-mga I have updated the PR to use `\t` literal.